src/coupled_driver.cpp

```diff
@@ -25,6 +25,7 @@
 #include <algorithm> // for copy
 #include <iomanip>
 #include <memory> // for make_unique
+#include <set>
```

Comment:
```suggestion
#include <map>
```

Reply:
fixed in https://github.com/enrico-dev/enrico/pull/131/commits/faecec3e5c3a4c81222f0da12657df095342d84e